universal_notifications/serializers.py

```diff
@@ -24,7 +24,6 @@ def get_configuration(self):
         request = self.context.get("request", None)
         if request and is_authenticated(request.user):
             return NotificationBase.get_mapped_user_notifications_types_and_categories(request.user)
-        return None
```

Comment:
returning None is default, but when you do it explicitly then when someone is reading the code he knows that you didn't just forgot about it